### PR TITLE
Relocate libautoupdate

### PR DESCRIPTION
**Pull request by lineargraph**

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/About.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class About {
 
     @ConfigOption(name = "Current Version", desc = "This is the SkyHanni version you are running currently")
     @ConfigVersionDisplay
-    public Void currentVersion = null;
+    public transient Void currentVersion = null;
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/About.java:14]: What does transient do in this context?

File: src/main/java/at/hannibal2/skyhanni/config/features/About.java [hunk 2]
```diff
@@ -26,6 +26,7 @@ public class About {
 
     @ConfigOption(name = "Used Software", desc = "Information about used software and licenses")
     @Accordion
+    @Expose
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/About.java:29]: Do we really need to save this into the config? we could refetch it from the jar on every startup anyway

File: src/main/java/at/hannibal2/skyhanni/config/features/About.java (hunk 1 above)
Comment (lineargraph) [reply src/main/java/at/hannibal2/skyhanni/config/features/About.java:14]: explicitly not serialized (supresses the warning for missing serialization)

File: src/main/java/at/hannibal2/skyhanni/config/features/About.java (hunk 2 above)
Comment (lineargraph) [reply src/main/java/at/hannibal2/skyhanni/config/features/About.java:29]: its just gonna save an empty object, and this is there to stop the warning as well

**Pull request merged**